common/enrollment/enroll.go

```diff
@@ -45,7 +45,7 @@ const outFlag = "out"
 func NewEnrollCommand() *cobra.Command {
```

Comment:
Should we just make the JWT the first positional argument to be consistent with other CLIs?

Reply:
I'm leaning toward yes. If it is required for all uses of the command, it should be a required positional.

Reply:
Change made. Also marked ziti-enroller as deprecated, so we can remove it in a few releases.